dev/scripts/repack.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+/* eslint-disable */
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ *
+ * Coverts Phaser 2 asset packs to Phaser 3
+ *
+ * Usage: node repack path-to-file
+ */
+// constants
+const brightRed = "\x1b[1m\x1b[31m";
+const brightGreen = "\x1b[1m\x1b[32m";
+const resetCli = "\x1b[0m";
+const fs = require("fs");
+//const path = require("path");
+
+// functions
+//const getNewPath = ({ dir, ext, name }) => ({ dir, ext, name: name + "_new" });
+const isP3Format = json => Object.keys(json).some(key => Boolean(json[key].prefix) || Boolean(json[key].files));
+const exitWithMessage = (color, text, code) => {
+    console.log(color, text);
+    console.log(resetCli);
+    process.exit(code);
+};
+
+const audio = entry => {
+    entry.url = entry.urls.find(value => /\.(mp3|mp4)$/i.test(value));
+    delete entry.urls;
+    delete entry.autoDecode;
+    return entry;
+};
+
+const isDefined = value => value !== undefined;
+const spriteParams = ["frameWidth", "frameHeight", "margin", "spacing", "startFrame", "endFrame"];
+
+const spritesheet = entry => {
+    const entries = spriteParams.map(param => [param, entry[param]]).filter(a => isDefined(a[1]));
+    entry.frameConfig = Object.fromEntries(entries);
+    [...spriteParams, "frameMax"].forEach(param => delete entry[param]);
+    return entry;
+};
```

Comment:
I feel like this could be a single array operation but I don't have enough coffee in my blood to follow up on that

Reply:
It's because "frameMax" isn't a thing in Phaser 3, do it needs deleting but doesn't need copying.